### PR TITLE
docs: corrected minute grammatical errors in README.md

### PR DESCRIPTION
**Pull request by alienishi**
### Description

I have corrected some minute grammatical errors in README.md

### Select what type of change this PR introduces:

1. [x] **Just code/docs improvement** (no functional change). 
2. [ ] **Bug fix** (non-breaking change which fixes an issue).
3. [ ] **New feature** (non-breaking change which adds functionality).
4. [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

### Update Waspc ChangeLog and version if needed
If you did a bug fix, new feature, or breaking change, that affects waspc, make sure you satisfy the following:
1. [ ] I updated [ChangeLog.md](https://github.com/wasp-lang/wasp/blob/main/waspc/ChangeLog.md) with description of the change this PR introduces.
2. [ ] I bumped waspc version in [waspc.cabal](https://github.com/wasp-lang/wasp/blob/main/waspc/waspc.cabal) to reflect changes I introduced, with regards to the version of the latest wasp release, if the bump was needed.

### CODE REVIEW

**Comment by alienishi** [comment]
@Martinsos @sodic 
Please review my PR and if according to the guidelines please do merge it :)

File: README.md [hunk 1]
```diff
@@ -46,11 +46,11 @@ app todoApp {
 
 route RootRoute { path: "/", to: MainPage }
 page MainPage {
-  authRequired: true, // Limit access to logged in users.
+  authRequired: true, // Limit access to logged-in users.
   component: import Main from "@client/Main.tsx" // Your React code.
 }
 
-query getTasks {
+query get tasks {
```
Comment (Martinsos) [inline README.md:53]: This should be modified.

File: README.md [hunk 2]
```diff
@@ -70,9 +70,9 @@ The rest of the code you write in React / Node.js / Prisma and just reference it
 
 <img width="800px" src="https://user-images.githubusercontent.com/1536647/231472285-126679e5-ecce-4cbb-8579-eb3cd9ba95bf.png"/>
 
-Given a simple .wasp configuration file that describes the high-level details of your web app, and .js(x)/.css/..., source files with your unique logic, Wasp compiler generates the full source of your web app in the target stack: front-end, back-end and deployment.
+Given a simple .wasp configuration file that describes the high-level details of your web app and .js(x)/.css/..., source files with your unique logic, the Wasp compiler generates the full source of your web app in the target stack: front-end, back-end, and deployment.
```
Comment (Martinsos) [inline README.md:73]: This doesn't read right.
I think we should `,` right after `...`, not the one you removed.

File: README.md [hunk 3]
```diff
@@ -95,17 +95,17 @@ This is the main repo of the Wasp universe, containing core code (mostly `waspc`
...
+The best place to start is to check out [waspc/](waspc/), where you can find detailed steps for first-time contributors + technical details about the Wasp compiler.
 
-Core of Wasp is built in Haskell, but there is also a lot of non-Haskell parts of Wasp, so you will certainly be able to find something for you!
+The core of Wasp is built in Haskell, but there are also a lot of non-Haskell parts of Wasp, so you will certainly be able to find something for you!
 
-Even if you don't plan to submit any code, just joining the discussion on discord [![Discord](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX) and giving your feedback is already great and helps a lot (motivates us and helps us figure out how to shape Wasp)!
+Even if you don't plan to submit any code, just join the discussion on discord [![Discord](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX) and giving your feedback is already great and helps a lot (motivates us and helps us figure out how to shape Wasp)!
```
Comment (Martinsos) [inline README.md:108]: I would revert `join` here to `joining`.

**Review by Martinsos — CHANGES_REQUESTED** [review]
Thanks for the contribution! I like most of the changes, but some of them need correction I think, I commented on those.

**Comment by alienishi** [comment]
@Martinsos 
Thanks for the suggestions!
I have made the changes according to your recommendations :)

**Review by Martinsos — APPROVED** [review]

**Comment by Martinsos** [comment]
Thanks @alienishi , merged!

**Pull request merged**